editor/code/src/client.ts

```diff
@@ -71,8 +71,6 @@ export function activate(context: ExtensionContext): void {
       if (goalPanel) goalPanel.dispose();
     }
 
-    window.showInformationMessage("Coq Language Server: starting");
```

Comment:
Changes to this file should have gone in a separate PR, right?

How is this message now displayed to the user?

Reply:
This message is no longer displayed, since the trace displays a more reliable "server start" message.